src/Configuration/Config.NewtonsoftJson/test/Microsoft.Extensions.Configuration.NewtonsoftJson.Tests.csproj

```diff
@@ -5,12 +5,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Config\test\Microsoft.Extensions.Configuration.Tests.csproj" />
-    <ProjectReference Include="..\..\testassets\Test.Common\Microsoft.Extensions.Configuration.Test.Common.csproj" />
```

Comment:
Why was this project deleted?

Reply:
The projects were migrated so we are removing them as much as possible but keeping a few files for tests.